### DISCUSSION TITLE
virus en repositorio falso, Tiktok-OSINT

### DISCUSSION THREAD

**Discussion by Delgado1969** [community/community#37792] [Other Feature Feedback, Questions, & Ideas] [answered]
Mucho cuidado el repositorio **https://github.com/FreeSoftwareOnGithubv2/Tiktok-OSINT** es falso e incluye un virus.
en realidad es una copia del código de https://github.com/Datalux/Osintgram/ al que se ha añadido un ejecutable malicioso.

**Comment by RubenGamarrarodriguez-tomtom** [comment]
Puedes reportarlo a GitHub

**Comment by otter-computer** [comment, marked as answer]
Hi @Delgado1969, thanks for the report! In future you can [report these instances directly](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to our Trust & Safety team to ensure they see them as soon as possible.

Thanks for keeping the GitHub community safe!

**Reply by zoro309** [reply]
I’m going out for

**Comment by Samy32377777** [comment]
I'm

**Comment by mohsen583** [comment]
Hi 
pip install -r requirements.txt 
Error install pakcage

**Comment by mohsen583** [comment]
Hi

hi

در تاریخ دوشنبه ۱۲ فوریه ۲۰۲۴،‏ ۲۱:۱۶ zoro309 ***@***.***>
نوشت:
